src/osgText/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ SET(COMPONENT_PKGCONFIG_DESCRIPTION "Text rendering library for OpenSceneGraph")
 
 if(OSG_TEXT_USE_FONTCONFIG AND Fontconfig_FOUND)
     MESSAGE(STATUS "osgText will be linked with FontConfig library")
-    list(APPEND TARGET_LIBRARIES Fontconfig::Fontconfig)
+    set(TARGET_EXTERNAL_LIBRARIES Fontconfig::Fontconfig)
```

Comment:
Why not:

     list(APPEND TARGET_EXTERNAL_LIBRARIES Fontconfig::Fontconfig)

Reply:
I thought it's empty, so I use set.
`list(APPEND ...)` is indeed better for future changes.